rsopt/pkcli/pack.py

```diff
@@ -16,35 +18,39 @@ def _get_local_modules(script_name):
 
     return file_list
 
+
 def _get_processing(job):
     # return pre/post processing scripts and local dependencies (if any)
     file_list = []
-    for attr in ['pre_process', 'post_process']:
-        if job.__getattribute__(attr):
-            file_name = job.__getattribute__(attr)[0]
+    for attr in ['preprocess', 'postprocess']:
+        if job.setup.get(attr):
+            file_name = job.setup.get(attr)[0]
             files = _get_local_modules(file_name)
             file_list.extend(files)
 
     return file_list
 
 
-def _get_file_list_from_model(model, code_name):
+def _get_file_list_from_model(model, code_name, input_file_path):
+    # Use same path as configuration file when packing
+    model_path = pathlib.Path(input_file_path).parents[0]
     # Get input and lattice if exists
-    file_list = [model['_SimData__source'].basename, ]
+    file_list = [model_path.joinpath(model['_SimData__source'].basename), ]
     try:
-        lattice_file = model._SimData__adapter._lattice_path('', model)
-        file_list.append(lattice_file)
+        lattice_file = model._SimData__adapter._lattice_path('', model).basename
```

Comment:
`if hasattr("_lattice_path")`?

Reply:
Much cleaner way to do this.
Also simplified the call a little while I was doublechecking the method 77f7d8873bdeff6872bc04ae5632b9cacd6b46ba.